stablehlo/dialect/StablehloOps.td

```diff
@@ -124,20 +124,27 @@ def StableHLO_IotaOp : StableHLO_Op<"iota", [Pure]> {
 def StableHLO_DynamicIotaOp: StableHLO_ShapedInterfaceOp<"dynamic_iota", [ConditionallySpeculatable, NoMemoryEffect]> {
   let summary = "DynamicIota operation";
   let description = [{
-    This operation is a work in progress, so it is not yet included in
-    the StableHLO specification: https://github.com/openxla/stablehlo/issues/8.
+    Fills a `result` tensor with values in increasing order starting from zero
+    along the `iota_dimension` dimension.
 
     Informally, this operation does the same thing as IotaOp except that the
     result shape is specified dynamically via `output_shape`:
     https://github.com/openxla/stablehlo/blob/main/docs/spec.md#iota
 
+    See:
+    https://github.com/openxla/stablehlo/blob/main/docs/spec.md#dynamic_iota
+
     Example:
     ```mlir
-    %0 = stablehlo.dynamic_iota %arg0, dim = 0 : (tensor<1xindex>) -> tensor<4xi32>
+    %output_shape = stablehlo.constant dense<[4, 5]> : tensor<2xi64>
+    %0 = stablehlo.dynamic_iota %output_shape, dim = 0 : (tensor<2xi64>) -> tensor<4x5xi64>
     ```
   }];
 
-  let arguments = (ins HLO_StaticDimensionTensor:$output_shape, I64Attr:$iota_dimension);
+  let arguments = (ins
+    I64Attr:$iota_dimension /*dynamic_iota_i1*/,
+    HLO_StaticDimensionTensor:$output_shape /*dynamic_iota_i2*/
+  );
```

Comment:
Why this change? StableHLO ops generally have their operands before their attributes.

Reply:
My [comment](https://github.com/openxla/stablehlo/pull/2270#discussion_r1589625143) wasn't clear. Michael's right; to clarify, `output_shape` should come first in the spec, that way, the `i1` and `i2` are also in order.

Reply:
reordered operand before attribute. Thank you.